### ISSUE TITLE
found bugs at Class CenitDataType , unlink , unable delete Data Type record.

### ISSUE THREAD

**Issue by kalmenchia** [openjaf/odoo-cenit#81]
Hi  , found potential bugs at data_definitions.py , 
class CenitDataType ,       
   
```
    @api.one
    def sync_rules(self):
        for trigger in self.triggers:
            trigger.sync()

     ...

    @api.one
    def unlink(self):
        flows = self._get_flows()

        res = super(CenitDataType, self).unlink()

        if res:
            self.sync_rules()
            #bugs --> self.sync_rules(flows)

        return res
```

where def  sync_rules(self): ... take no other params then SELF .

**Comment by sanchojaf** [comment]
Thanks @kalmenchia for report the issue!!

**Comment by maryguirola** [comment]
Fixed! #82
